packages/flutter/test/cupertino/bottom_tab_bar_test.dart

```diff
@@ -327,6 +327,48 @@ Future<void> main() async {
     expect(tester.getSize(find.byType(CupertinoTabBar)).height, 90.0);
   });
 
+  testWidgets('Set height', (WidgetTester tester) async {
```

Comment:
Nit: This might be a bit clearer 
```suggestion
  testWidgets('Set custom height', (WidgetTester tester) async {
```